### PR TITLE
feat: show content in main el

### PR DESCRIPTION
**Pull request by binsarjr**
* more like mpa
* seo++

### CODE REVIEW

**Comment by binsarjr** [comment]
## Before
```html
<main id="app"></main><!---->
```

## After
```html
<main id="app"><div><p>#{from_server2}</p>
	<table><thead><tr><th>Col1</th>
			<th>Col2</th></tr></thead>
		<tbody><tr><td class="svelte-o8xmoi">A</td><td class="svelte-o8xmoi">1</td>
</tr><tr><td class="svelte-o8xmoi">B</td><td class="svelte-o8xmoi">2</td>
</tr><tr><td class="svelte-o8xmoi">C</td><td class="svelte-o8xmoi">3</td>
</tr></tbody></table></div>

<hr>
<a href="[/](view-source:http://127.0.0.1:5500/)">/</a> | 
<a href="[/page2](view-source:http://127.0.0.1:5500/page2)">/page2</a> | 
<a href="[/page1/subpage.html](view-source:http://127.0.0.1:5500/page1/subpage.html)">/page1/subpage.html</a> |
<a href="[/page1/subpage3](view-source:http://127.0.0.1:5500/page1/subpage3)">/page1/subpage3</a></main><script>document.getElementById('app').innerHTML=''</script><!---->
```

File: build.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+require('svelte/register')
```
Comment (kokizzu) [inline build.js:1]: what does it do?

Comment (binsarjr) [reply build.js:1]: [Render svelte to html](https://svelte.dev/docs#run-time-svelte-register)

**Comment by kokizzu** [comment]
looks good but haven't test it to make sure it doesn't break anything

**Pull request merged**